spec/models/json_document_importer_spec.rb

```diff
@@ -134,17 +130,16 @@
       }
 
       context 'and creates claims' do
-        let(:subject) { JsonDocumentImporter.new(json_file: exported_claims, schema_validator: schema_validator, api_key: 'test_key') }
+        subject(:importer) { JsonDocumentImporter.new(json_file: exported_claims, schema_validator: schema_validator, api_key: 'test_key') }
 
         it 'from valid hashes' do
-          expect(subject).to receive(:create_claim).exactly(2).times
-          subject.import!
-          expect(subject.errors.blank?).to be true
+          expect(importer).to receive(:create_claim).exactly(2).times
+          importer.import!
+          expect(importer.errors.blank?).to be true
```

Comment:
**[improvement/nit pick/not a blocker]** 💭 Whenever you have `<object>.<method_name>?` I believe **RSpec** support a nicer syntax:

```ruby
expect(importer.errors).to be_blank
# or
expect(importer.errors).not_to be_present
```